tldw_Server_API/app/api/v1/endpoints/audio.py

```
@@ -90,7 +99,147 @@
+async def _stream_tts_to_websocket(
+    *,
+    websocket: WebSocket,
+    speech_req: OpenAISpeechRequest,
+    tts_service: Any,
+    provider: Optional[str],
+    outer_stream: Optional[Any],
+    reg: Any,
+    route: str,
+    component_label: str,
+    voice_to_voice_start: Optional[float] = None,
+    error_handler: Optional[Callable[[Exception], Awaitable[None]]] = None,
```

Ruff (0.14.7)

Undefined name `Callable`

(F821)